src/custom/state/sentry/updater/index.ts

```diff
@@ -3,44 +3,71 @@ import * as Sentry from '@sentry/browser'
 
 import useIsWindowVisible from 'hooks/useIsWindowVisible'
 import { useSwapState } from 'state/swap/hooks'
-import { SupportedChainId } from 'constants/chains'
 import { useCurrency } from 'hooks/Tokens'
 import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useAppSelector } from 'state/hooks'
+
+function _getSentryChainId(appChainId: null | number, chainId: number | undefined) {
+  const connectedChainId = chainId
+  const appChainIdFull = appChainId ?? undefined
+
+  // app and connected chains the same
+  if (appChainIdFull === connectedChainId) {
+    return appChainIdFull
+    // appChain but no connected chain
+  } else if (connectedChainId === undefined) {
+    return `${appChainIdFull} - DISCONNECTED`
+    // connected chain but doesn't match app chain, use connected chain
```

Comment:
This the comments apply to the next block or current block?
If next, can you move the comments down